CSS/style.css

```diff
@@ -505,3 +505,12 @@ textarea:hover {
   font-weight: 500;
   line-height: 20px;
 }
+
+.message {
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  color: white;
+  text-align: center;
+  padding-bottom: 20px;
+}
```

Comment:
**[Optional]**
- [ ] It would be nice if you add the `color: red;` which is a standard color for the errors and make your form human-friendly :blush: